atef/widgets/config.py

```diff
@@ -2210,3 +2213,220 @@ class LessOrEqualWidget(CompMixin, GtLtBaseWidget):
+        - Set up the dynamic behavior of the visualization
+        """
+        # Line edits and visualization
+        for ident in ('low', 'high', 'warn_low', 'warn_high'):
+            line_edit = getattr(self, f'{ident}_edit')
+            value_obj = getattr(self.bridge, ident)
+            # Copy all changes to the visualization labels
+            label = getattr(self, f'{ident}_label')
+            line_edit.textChanged.connect(label.setText)
+            # Trigger the visualization update on any update
+            value_obj.changed_value.connect(self.update_visualization)
+            # Standard setup and initialization
+            setup_line_edit_all(
+                line_edit=line_edit,
+                value_obj=value_obj,
+                from_str=float,
+                to_str=str,
+                minimum=100,
+                buffer=15
+            )
+        # Checkbox
+        self.bridge.inclusive.changed_value.connect(
+            self.inclusive_check.setChecked
+        )
+        self.bridge.inclusive.changed_value.connect(
+            self.update_visualization
+        )
+        self.inclusive_check.clicked.connect(self.bridge.inclusive.put)
+        self.inclusive_check.setChecked(self.bridge.inclusive.get())
+        # Symbols
+        self.bridge.inclusive.changed_value.connect(self.update_symbols)
+        self.update_symbols(self.bridge.inclusive.get())
+        # One additional visual update on inversion
+        self.bridge.invert.changed_value.connect(self.update_visualization)
+        # Make sure this was called at least once
+        self.update_visualization()
+
+    def update_symbols(self, inclusive: bool) -> None:
+        """
+        Pick the symbol type based on range inclusiveness.
+
+        Use the less than symbol if not inclusive, and the the
+        less than or equals symbol if inclusive.
+
+        Parameters
+        ----------
+        inclusive : bool
+            True if the range should be inclusive and False otherwise.
+        """
+        if inclusive:
+            symbol = '≤'
+        else:
+            symbol = '<'
+        for index in range(1, 5):
+            label = getattr(self, f'comp_symbol_label_{index}')
+            label.setText(symbol)
+
+    def resizeEvent(self, *args, **kwargs) -> None:
+        """
+        Override resizeEvent to update the visualization when we resize.
+        """
+        self.update_visualization()
+        return super().resizeEvent(*args, **kwargs)
+
+    def update_visualization(self, *args, **kwargs):
+        """
+        Make the visualization match the current data state.
+        """
+        # Cute trick: swap red and green if we're inverted
+        if self.bridge.invert.get():
+            green = self.red
+            red = self.green
+        else:
+            green = self.green
+            red = self.red
+        yellow = self.yellow
+        self.left_red_line.penColor = red
```

Comment:
I think the vertical lines could use a larger pen width to make them more visible

Reply:
Easy to adjust

Reply:
Thicker lines: 
![image](https://user-images.githubusercontent.com/10647860/166571623-75946ad7-1cd8-49a4-8092-c39fa71a4db8.png)